db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create(email: 'test@test.com', name: 'Allison Higgenbothan', password: '1234')
+User.create(email: 'test2@test.com', name: 'Aln Higgenbothan', password: '1234')
+User.create(email: 'test3@test.com', name: 'Allison ggenbothan', password: '1234')
+User.create(email: 'test4@test.com', name: 'Allin Higgenbothan', password: '1234')
+User.create(email: 'test5@test.com', name: 'Allison Higgothan', password: '1234')
+User.create(email: 'test6@test.com', name: 'Ason Hiothan', password: '1234')
+
+Post.create(author_id: '1', title: 'Test', body: 'testing')
```

Comment:
You shouldn't hardcode the author_id here, as you don't actually know what the `id`s generated on lines 9-14 will be.  You can only be sure they are 1-5 *if* `rake db:seed` is run immediately after the table is created.

You should store the users in variables and then create the posts by passing in `author: user1` or similar.

Also, I'm fairly certain that running `rake db:seed` will fail, since you're specifying the Users' passwords but not the password_confirmations.